src/Razor/test/testapps/Directory.Build.props

```diff
@@ -1,5 +1,7 @@
 <Project>
   <Import Project="Before.Directory.Build.props" Condition="Exists('Before.Directory.Build.props')" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\..\eng\Versions.props" Condition="'$(SolutionRoot)' == ''" />
```

Comment:
Hmmm, why both this and the one below?

Reply:
Because the condition. One is true when we run `build.cmd` locally. The other is true when this is run as a test.

Reply:
Errm, why not import this after `SolutionRoot` is calculated?